public/config/configuration.json

```diff
@@ -10,6 +10,7 @@
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
   "LEGAL_API_VERSION": "/api/v1",
+  "NAICS_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/naics",
```

Comment:
New variable to make it easy to change to a new endpoint/service in the near future.

Reply:
I asked Patrick to add this to Dev/Test/Prod.

Reply:
> I asked Patrick to add this to Dev/Test/Prod.

And actually, I asked him to show me how, so I did it myself 😃

Reply:
Please note that the config keys have changed from the above (in commit 4).